index.html

```diff
@@ -12,7 +12,16 @@
 <main>
   <header>
     <nav>
-      <div class="logo"><p>John Doe</p></div>
+      <div class="logo"><p>Tessy</p></div>
+      <ul class="menu-bar">
+        <li><a href="#frame" class="head-link">Portfolio</a></li>
+        <div class="indicator"></div>
```

Comment:
- [optional] please there is this extra margin right on your mobile version. please look into that
![Screenshot (656)_LI](https://user-images.githubusercontent.com/65298048/206654886-25bc256c-e759-45fa-b6f8-90d57f1a59ff.jpg)